cmake/CTestCostData.txt

```diff
@@ -0,0 +1,132 @@
+rnp_tests-hash_test_success 3 0.0164573
```

Comment:
Do I correctly understand that we should update this file with each new test added?

Reply:
It's not necessarily beneficial to update it on every added test. It's more useful to update it occasionally or particularly if adding a long-running test. Tests that aren't listed here will default to a COST of 0 (likely to start running last).

I need to add this to the dev guide.